src/packages/utils/package.json

```diff
@@ -44,7 +44,10 @@
     "babel-loader": "8.2.2",
     "babel-plugin-transform-class-properties": "6.24.1",
     "cross-env": "7.0.3",
+    "css-loader": "5.0.2",
     "file-loader": "6.2.0",
+    "mini-css-extract-plugin": "1.3.8",
+    "sass-loader": "11.0.1",
```

Comment:
do we need all those?

Reply:
I took these from excalidraw package.

Reply:
technically, for this feature we only need `sass-loader`, but `css-loader` makes sense. No clue what `mini-css-extract-plugin` is for, would need to ask @ad1992. But I guess we can update it later.

Reply:
I'll try removing it and check if build passes?

Reply:
Even in `excalidraw` package `mini-css-extract-plugin` is not found anywhere in webpack config.

Reply:
`mini-css-extract-plugin` is not needed here. you can remove it, it basicially to extract CSS to separate file which is not needed here.

Reply:
Got it! Removed it :)